developers/academy/zero_to_mvp/101_hello_weaviate/15_overview_vectors.mdx

```diff
@@ -9,56 +9,57 @@ import ReactPlayer from 'react-player/lazy'
-
-Each vector representation of such "meaning" is called a vector embedding.
+Now, imagine having hundreds, or even thousands, of these dials. That’s how vectors are used to represent meaning. Modern machine learning models such as GPT-x, or those used with Weaviate, use vectors to represent some "essence", or "meaning" of objects. This can be done for any object type, such as text, code, images, videos and more.
 
 ## <i class="fa-solid fa-square-chevron-right"></i>&nbsp;&nbsp;Vector embeddings in Weaviate
 
 <!-- <img src={imageUrl} alt="Image alt" width="75%"/> -->
 
-Weaviate enables vector searches by indexing and storing data objects and corresponding vector embeddings from machine learning models.
+The vector representation of an object's meaning is called a "vector embedding".
+
+Weaviate enables vector searches by indexing and storing data objects and their corresponding vector embeddings. The vector embeddings come from machine learning models.
 
-In plain terms, Weaviate processes and organizes your data in such a way that objects can be retrieved based on their similarity to a query. In order for it to perform these tasks at speed, Weaviate does two things that traditional databases do not. They are:
+In plain terms, Weaviate processes and organizes your data in such a way that objects can be retrieved based on their similarity to a query. To perform these tasks at speed, Weaviate does two things that traditional databases do not. Weaviate:
 
-- Quantifying similarity, and
-- Indexing vector data
+- Quantifies similarity
+- Indexes vector data
 
-These aspects enable Weaviate to do what it does.
+These operations enable Weaviate to do what it does.
 
 ### <i class="fa-solid fa-chalkboard"></i>&nbsp;&nbsp;Quantifying similarity
 
 As we've mentioned, vector searches are similarity-based, but what does that actually mean? How do we determine that two pieces of data are "similar"? What does it mean for two pieces of text, two images, or two objects in general, to be similar?
 
 This is a relatively simple idea that is actually incredibly interesting and intricate once we start to dive into the details.
 
-But for now, you should know that machine learning (ML) models are key to this whole process. Similar models to those that allows clever text generation from prompts power vector searches. Instead of generating new text, here these models capture "meaning" of pieces of text or other media. We will cover this in more detail later on.
+But for now, you should know that machine learning (ML) models are the key to this whole process. Some ML models use the prompts that power vector searches to generate text responses. Instead of generating new text, the Weaviate models capture the "meaning" of text or other media. We will cover this in more detail later on.
 
```

Comment:
I think this should read:

`The ML models that power vector searches share similarities with those that generate text responses from prompts.` 

The models that perform vector searches are not the same as those that perform text generation. The models are not Weaviate models either (as in, we don't build models). We do use them, however. 

So overall, how about:

```
But for now, you should know that machine learning (ML) models are the key to this whole process. The ML models that power vector searches share similarities with those that generate text responses from prompts. Instead of generating new text, these (vectorizer) models capture the "meaning" of text or other media. We will cover this in more detail later on.
```